bioio/tests/conftest.py

```diff
@@ -36,10 +36,12 @@
 
 
 @pytest.fixture
-def sample_text_file(tmp_path: pathlib.Path) -> pathlib.Path:
+def sample_text_file(
+    tmp_path: pathlib.Path,
+) -> typing.Generator[pathlib.Path, None, None]:
     example_file = tmp_path / "temp-example.txt"
     example_file.write_text("just some example text here")
-    return example_file
+    yield example_file
```

Comment:
Unrelated to main changeset, this is just to make the tests faster by only creating the file once since they seemed slow